hknweb/tours/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.8 on 2020-03-25 05:37
+
+import datetime
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DepTour',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(default='', max_length=255)),
+                ('email', models.EmailField(default='', max_length=255)),
+                ('confirm_email', models.EmailField(default='', max_length=255)),
```

Comment:
I don't think both of these need to be persisted in the database, after all their values should match to be able to submit the form so it's just duplicate information by the time it gets into the database itself, right?